webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
+const path = require('path')
+//const webpack = require('webpack')
```

Comment:
Remove unused `require`.

Reply:
This was removed in 5d604c5f83ec276107c7331b4c57c03e12f1a4a9